tools/gulptasks/unsorted/uploadS3.js

```diff
@@ -0,0 +1,240 @@
+const semver = require('semver');
```

Comment:
If it is used only as a lib, please move it to `tools/gulptasks/lib`.

Reply:
Fixed.